src/machines/lesson-machine.ts

```diff
@@ -43,18 +43,18 @@ export type PlayerStateEvent =
   | {type: 'ADD_NOTE'}
   | {type: 'done.invoke.fetchLessonDataService'; data: any}
 
-interface PlayerContext {
+interface LessonContext {
   lesson: any
   viewer: any
 }
 
-export const playerMachine = Machine<
-  PlayerContext,
-  PlayerStateSchema,
-  PlayerStateEvent
+export const lessonMachine = Machine<
+  LessonContext,
+  LessonStateSchema,
+  LessonStateEvent
```

Comment:
Old player machine becomes more of a lesson machine responsible for toggling overlays and stuff. Not much else had to change here.